features/networking/sgw-lgw-migration.feature

```diff
@@ -0,0 +1,123 @@
+Feature: SGW<->LGW migration related scenarios
+  
+  # @author anusaxen@redhat.com
+  # @case_id OCP-47561
+  @4.10
+  @admin
+  @destructive
+  @network-ovnkubernetes
+  @vsphere-ipi
+    Scenario: [SDN-2290] SGW <-> LGW migration scenario	for vsphere platform
```

Comment:
Scenario name should be one less level indention.

Reply:
Yes. Thanks @liangxia PTAL now